### PR TITLE
Bump version to 8.0.0

### PR DESCRIPTION
**Pull request by benilovj**
/cc @fofr

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# 8.0.0
+
+* Switch from JS custom variables to HTML meta tags
+
+  Slimmer now appends page metadata as meta tags instead of setting Google
+  custom variables within a script tag. The Google-specific implementation
+  details have been removed. Any apps that need to report analytics events
+  will require additional Javascript that reads the meta tags and sends the relevant
+  data to the analytics platform.
```
Comment (dsingleton) [inline CHANGELOG.md]: Worth cross linking this to the specific JS/docs required for this switch? Good for someone (less familiar with the change) having to upgrade slimmer.

Comment (benilovj) [inline CHANGELOG.md]: Good call - updated and force pushed (9dcfcb7)

**Comment by edds** [comment]
I know this has now been merged, but I am not entirely sure that this was a major change. The API for the gem hasn't actually changed and major changes should indicate there was a breaking API change in this gem. This change will have ramifications for our frontend code in static as it lets us remove an element which was otherwise being used by slimmer and if we don't update static will will stop reporting data we would have otherwise been reporting. I could be wrong here though, but that is my current thinking.

**Comment by benilovj** [comment]
I made this a breaking change because after upgrade, the app stops sending certain analytics events that it was sending before, potentially creating a hole in the analytics data.

**Pull request merged**